src/common/calendar/TimePickerCalendar.js

```diff
@@ -106,6 +124,26 @@ class TimePickerCalendar extends Component {
     this.onChange(selectable);
   }
 
+  onDatesRender = (info) => {
+    const { viewType, header } = this.state;
+    let view = info.view.type;
+    let headerConfig = header;
+
+    if (window.innerWidth < 768) {
```

Comment:
So that would apply for small screens, not necessarily mobile devices. Are we ok with that?

Reply:
I think so, screen under 768 width is _mostly_ considered as mobile devices